### DISCUSSION TITLE
Items existing in multiple locations

### DISCUSSION THREAD

**Discussion by dewoodruff** [sysadminsmedia/homebox#56] [General]
Hi! I found Homebox on the day it was switching ownership and it looks like the simplest, most straight forward option for my basic use case of a freezer and pantry inventory. I just want to know how many jars of apple sauce are in the basement pantry and how many quarts of berries are in the upstairs freezer. 

I'm trying to figure out the best structure for locations, labels, and parent items to handle an item in multiple locations. So, 1 quart of berries in the upstairs freezer readily accessible, and 3 quarts in the basement freezer. About the only thing I can come up with is ignoring Locations and just titling items as "Berries - upstairs freezer" and "Berries - downstairs freezer" so I can have different quantities for each.

Any other creative ideas on how to make this work cleanly? Thanks! And thanks for sysadminsmedia picking this up.

**Comment by shielnic** [comment]
Have you tried Grocy for food inventory https://github.com/grocy/grocy. I use Grocy for my food and Homebox for my assets.

**Reply by dewoodruff** [reply]
Thanks, yeah I have played with it but it was more complicated and detailed than I really wanted or could expect my family to work with.

**Reply by tankerkiller125** [reply]
Quite honestly, I don't think that Homebox is really the correct solution for managing things like food.

Or if you did want to, separate assets for different locations.

For some context here, I work with manufacturing companies at work (ERP, which involves a lot of inventory and BOMs) and over the many decades that the company has existed the consultants and developers I work with have found that it's best to just keep each location separate when it comes to inventory with separate IDs (including and especially consumables). Even if they are the same thing, they get treated as separate.

Now of course, we work with much more complex and crazy software, so it has mechanisms to report all of those separate assets into one number via various mechanisms, but its primary state is separate.

I do kind of think it would be neat though if we could find a way to link two separate items together in different locations in such a way that you can tell Homebox it's the same item. And then have a view on page that shows you the inventory of the item in other locations? That seems like a decent idea.

**Reply by dewoodruff** [reply]
Thanks! I see Homebox isn't designed for food because food management often comes with expiration dates and specific quantities. I know everybody has different use cases but I'm not interested in that level of detail. I simply want to know how many quarts of berries are in the freezer or how many jars of tomatoes are left, and where they are. Garden season is upon us :)

Knowing now that people use Homebox for insurance purposes and track items in detail, I suppose it is not likely that people have the exact same item in multiple places in their home. Maybe there are 2 TVs but they're probably different models and definitely different serial numbers. So I see my use case doesn't directly translate to Homebox. The linking of the items you mentioned could be a solution that would be good enough though, without me just relying on names of items.

**Reply by tomyan112** [reply]
Create multiple items for the food. Use the search function, it would show the number of results so you would know how many of the items you have left.
